### DISCUSSION TITLE
v2.2 Road Map

### DISCUSSION THREAD

**Discussion by jwildfire** [SafetyGraphics/safetyGraphics#710] [General]
Remaining tasks for upcoming release of safetyGraphics v2.2 and associated libraries

- **safetyGraphics v2.2**
  - [ ] clear checks
  - [ ] shinyapps.io refactor
  - [ ] Update vignettes/readme
- **safetyCharts v0.4**
   -  Improve documentation on adding charts
   -  Add new domains
   - Clear Checks 
- **safetyProfile v1.0**
- **nepExplorer v1.0**
- **QTexplorer v1.0**
- **AEVolcanoPlot v1.0**
- **Website refactor** - update group homepage and add list of supported packages/charts

**Comment by samussiah** [comment]
@nandriychuk - v2.2 TODOs

**Reply by nandriychuk** [reply]
Suggestion from @xni7:
Consider creating consolidated vignettes for SafetyGraphcs so that users can access the individual chart vignettes  under one umbrella.
